src/ebonite/build/docker.py

```diff
@@ -70,12 +69,18 @@ def get_uri(self) -> str:
             uri = '{}/{}'.format(self.registry.get_host(), uri)
         return uri
 
-    def to_core_image(self) -> Image:
-        return Image(self.get_uri(), params=serialize(self))
 
-    @staticmethod
-    def from_core_image(image: Image) -> 'DockerImage':
-        return deserialize(image.params, DockerImage)
+class DockerContainer(RuntimeInstance.Params):
+    # FIXME `ports_mapping` keys could be effectively `int` or `str` but
```

Comment:
just added support for int keys t opyjackson, try pyjackson==0.0.25

Reply:
ok, hope it helps

Reply:
it did